lang/csharp/src/apache/main/CodeGen/CodeGen.cs

```diff
@@ -148,6 +145,20 @@ protected virtual CodeNamespace addNamespace(string name)
             return ns;
         }
 
+        /// <summary>
+        /// Adds a namespace object for the given name into the dictionary if it doesn't exist yet.
+        /// </summary>
+        /// <param name="name">name of namespace.</param>
+        /// <returns>
+        /// Code Namespace.
+        /// </returns>
+        /// <exception cref="System.ArgumentNullException">name - name cannot be null.</exception>
+        [Obsolete("Use AddNamespace. This call will be deprecated in a future release.")]
```

Comment:
Please see https://github.com/apache/avro/pull/1484#discussion_r796496760
I am not native English speaker, so I might be wrong !

Reply:
Updated messaging